data-prepper-plugins/opensearch/src/main/resources/index-template/metrics-otel-v1-index-template.json

```diff
@@ -3,50 +3,103 @@
   "template": {
     "mappings": {
       "date_detection": false,
+      "_source": {
+        "enabled": true
+      },
       "dynamic_templates": [
         {
-          "resources_map": {
+          "long_resource_attributes": {
             "mapping": {
-              "type": "keyword"
+              "type": "long"
+            },
+            "path_match": "resource.attributes.*",
+            "match_mapping_type": "long"
+          }
+        },
+        {
+          "double_resource_attributes": {
+            "mapping": {
+              "type": "double"
+            },
+            "path_match": "resource.attributes.*",
+            "match_mapping_type": "double"
+          }
+        },
```

Comment:
This one is now missing the string attributes for resources, scope and plain.

Reply:
Thanks @KarstenSchnitter. Will do a separate PR for this.

Reply:
These are existing mappings. Not focusing on these in this PR, anyways.